README.md

```diff
@@ -101,15 +101,15 @@ Contact: __pu2clr@gmail.com__.
 
 ## Why do skilled developers generally use libraries?
 
-Technically, no Arduino application needs a library.  In some cases, do not use libraries may be a nice choice. However, why do skilled developers generally use libraries?
```

Comment:
I changed this file to make it easier and more natural to read. My change is not perfect, but it is more readable to a native English speaker than before.